components/public-api/gitpod/v1/prebuild.proto

```diff
@@ -83,15 +97,20 @@ message PrebuildStatus {
   string message = 3;
 }
 
+enum Phase {
```

Comment:
We could name it with `PrebuildPhase`?

Another FYI, we could also use `PrebuildPhase.Phase` inside `Filter`

Reply:
did not know you can use these nested enums from outside. cool stuff